topo_processor/metadata/loaders/metadata_loader_tiff.py

```diff
@@ -14,8 +16,19 @@ class MetadataLoaderTiff(MetadataLoader):
     def is_applicable(self, item: Item) -> bool:
         return is_tiff(item.path)
 
-    def add_metadata(self, item: Item) -> None:
-        # TODO use GDAL to load metadata from the tiff file if it exists
+    async def add_metadata(self, item: Item) -> None:
+        with rasterio.open(item.path) as tiff:
+            # projection
+            if tiff.crs:
```

Comment:
whats the diference between tiff.crs and tiff.profile['crs']

Reply:
they are the same.

Reply:
they are the same. 

`print(tiff.crs is tiff.profile['crs'])  # True`